corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -11,170 +12,140 @@
+                        {% case "missing parent tag" %}
                             A subcase is referencing a parent case tag that does not exist: "{{ error.case_tag }}"
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "subcase repeat context" %}
+                        {% case "subcase repeat context" %}
                             The subcase "{{ error.case_tag }}" is in a different repeat context to its parent
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "circular ref" %}
+                        {% case "circular ref" %}
                             A circular reference was detected in the hierarchy of subcase "{{ error.case_tag }}"
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "auto select key" %}
+                        {% case "auto select key" %}
                             The auto-select case action is missing the "{{ error.key_name }}" value
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "auto select source" %}
+                        {% case "auto select source" %}
                             The auto-select case action is missing the "{{ error.source_name }}" value
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "auto select case ref" %}
+                        {% case "auto select case ref" %}
                             The case tag referenced in the auto-select expression of "{{ error.case_tag }}" was not found
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "no case type in action" %}
+                        {% case "no case type in action" %}
                             The form action "{{ error.case_tag }}" does not have a case type selected
                             {% if not not_actual_build %}in form {% include "app_manager/partials/form_error_message.html" %}{% endif %}
-                        {% endif %}
-                        {% if error.type == "no case type in action" %}
+                        {% case "filtering without case" %}
```

Comment:
This error was [originally added](https://github.com/dimagi/commcare-hq/commit/bc30fcf16bb609a3f9e6f0d349c4c6cdb4c4ff31) with the wrong type. The new `case` tag will prevent that from happening in the future.